src/test/regress/expected/multi_join_order_additional.out

```diff
@@ -154,10 +156,12 @@ EXPLAIN SELECT count(*) FROM lineitem, orders
 	WHERE (l_orderkey = o_orderkey AND l_quantity > 5)
 	OR (l_orderkey = o_orderkey AND l_quantity < 10);
 LOG:  join order: [ "lineitem" ][ local partition join "orders" ]
-                         QUERY PLAN                         
-------------------------------------------------------------
- explain statements for distributed queries are not enabled
-(1 row)
+                                QUERY PLAN                                
+--------------------------------------------------------------------------
+ Aggregate  (cost=0.00..0.00 rows=0 width=0)
+   ->  Custom Scan (Citus Task-Tracker)  (cost=0.00..0.00 rows=0 width=0)
+         explain statements for distributed queries are not enabled
```

Comment:
Hm, I suppose. The three words I have trouble with discerning are _query_, _statement_, and _command_. Clearly _queries_ return results. The latter two, I'm not certain about, but I think I favor _command_ for some reason. _Statement_ seems something like _a = 5_ or _if (error) { return; }`. I guess in some languages those are also _expressions_. I'd have to check PostgreSQL's error messages to know what they favor, but my gut instinct is they also like _command_, even though the SQL standard divides things into _statements_ and _queries_.

This is obviously pretty nitpicky, but it was just something I noticed about that error message which bothered me (and is not even part of this change, so ignore I guess).